.snyk

```diff
@@ -28,9 +28,5 @@ ignore:
     - '*':
         reason: >-
           Code is only run client-side. No risk of arbitrary file upload.
-  SNYK-GOLANG-GOPKGINYAMLV2-2840885:
```

Comment:
This was revoked. https://security.snyk.io/vuln/SNYK-GOLANG-GOPKGINYAMLV2-2840885